suites/quincy/rbd/rbd_persistent_write_back_cache.yaml

```diff
@@ -0,0 +1,146 @@
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: RHCS cluster deployment using cephadm
+      destroy-clster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+#  Test cases to be executed
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        node: node6
+        install_packages:
+          - ceph-common
+          - fio
+        copy_admin_keyring: true
+      desc: Configure client node
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+      polarion-id: CEPH-83573758
+
+  - test:
+      abort-on-fail: true
+      config:
+        level: global                        # PWL at global
+        cache_file_size: 1073741824          # 1 GB
+        rbd_persistent_cache_mode: ssd       # "ssd" or "rwl" on pmem device
+        client: node6
+        drive: /dev/sdd
+        cleanup: true
+        rep-pool-only: True
+        rep_pool_config:
+          pool: pool1
+          image: image1
+          size: 10G
+        fio:
+          image_name: image1
+          pool_name: pool1
+          runtime: 120
+      desc: PWL validation at global level
+      destroy-cluster: false
+      module: rbd_persistent_write_back_cache.py
+      name: RBD Persistent Cache - Global level configuration
+      polarion-id: CEPH-83574707
+
+  - test:
+      name: RBD Persistent Cache Image and Pool level configuration
+      desc: PWL validation at pool and image level.
+      module: test_parallel.py
+      polarion-id: CEPH-83574709
+      abort-on-fail: true
+      parallel:
+      - test:
+          config:
+            level: pool                          # PWL at Pool level
+            cache_file_size: 2147483648          # 2 GB
+            rbd_persistent_cache_mode: ssd       # "ssd" or "rwl" on pmem device
+            client: node6
+            drive: /dev/sdd
+            cleanup: true
+            rep-pool-only: True
+            rep_pool_config:
+              pool: pool2
+              image: image2
+              size: 20G
+            fio:
+              image_name: image2
+              pool_name: pool2
+              runtime: 120
+          desc: PWL validation at pool level
+          destroy-cluster: false
+          module: rbd_persistent_write_back_cache.py
+          name: RBD Persistent Cache - Pool level configuration
+          polarion-id:
```

Comment:
polarion id is missing, please add that

Reply:
Oh, need to take that away. the actual test case polarion id is added at parallel.